Lib/test/test_ntpath.py

```diff
@@ -107,13 +108,13 @@ def test_splitdrive(self):
         tester('ntpath.splitdrive("//conky/mountpoint/foo/bar")',
                ('//conky/mountpoint', '/foo/bar'))
         tester('ntpath.splitdrive("\\\\\\conky\\mountpoint\\foo\\bar")',
-            ('', '\\\\\\conky\\mountpoint\\foo\\bar'))
+            ('\\\\\\conky', '\\mountpoint\\foo\\bar'))
         tester('ntpath.splitdrive("///conky/mountpoint/foo/bar")',
-            ('', '///conky/mountpoint/foo/bar'))
+            ('///conky', '/mountpoint/foo/bar'))
         tester('ntpath.splitdrive("\\\\conky\\\\mountpoint\\foo\\bar")',
-               ('', '\\\\conky\\\\mountpoint\\foo\\bar'))
+               ('\\\\conky', '\\\\mountpoint\\foo\\bar'))
```

Comment:
In practice, `"\\\\conky\\\\mountpoint\\foo\\bar"` is a valid path that the system will open as `r"\\conky\mountpoint\foo\bar"`. If that's the working directory, then opening the rooted path `"\spam\eggs"` will open `r"\\conky\mountpoint\spam\eggs"`. So the drive is `"\\\\conky\\\\mountpoint"`. It would be better if `ntpath.join()` could get this right. But the result was wrong before, too, just in a different way.

Reply:
As with above, it'd be better if the OS gets it right. We spend far too long chasing these edge cases ourselves.